cli/docker_validator.py

```diff
@@ -0,0 +1,227 @@
+import argparse
+import csv
+import logging
+import os
+import subprocess
+import time
+
+logging.basicConfig(format='%(asctime)s %(message)s', datefmt='%m/%d/%Y %I:%M:%S %p', level='INFO')
+
+container_image = 'eva_sub_cli'
+container_validation_dir = '/opt/vcf_validation'
+container_validation_output_dir = '/opt/vcf_validation/vcf_validation_output'
+
+
+def run_command_with_output(command_description, command, return_process_output=True,
+                            log_error_stream_to_output=False):
+    process_output = ""
+
+    logging.info(f"Starting process: {command_description}")
+    logging.info(f"Running command: {command}")
+
+    stdout = subprocess.PIPE
+    # Some utilities output non-error messages to error stream. This is a workaround for that
+    stderr = subprocess.STDOUT if log_error_stream_to_output else subprocess.PIPE
+
+    with subprocess.Popen(command, stdout=stdout, stderr=stderr, bufsize=1, universal_newlines=True,
+                          shell=True) as process:
+        for line in iter(process.stdout.readline, ''):
+            line = str(line).rstrip()
+            logging.info(line)
+            if return_process_output:
+                process_output += line + "\n"
+        if not log_error_stream_to_output:
+            for line in iter(process.stderr.readline, ''):
+                line = str(line).rstrip()
+                logging.error(line)
```

Comment:
Might be obvious but this will not handle a process that write lots output to stderr. It will block the process as soon as the cache is if full.

Reply:
I know it's the same as https://github.com/EBIvariation/eva-common-pyutils/blob/master/ebi_eva_common_pyutils/command_utils.py